### PR TITLE
Update s3 bucket creation script

### PR DESCRIPTION
**Pull request by ryley-o**
## Description of the change

Incorporate required updates, as described here:

https://aws.amazon.com/blogs/aws/heads-up-amazon-s3-security-changes-are-coming-in-april-of-2023/

### CODE REVIEW

**Review by jakerockland — APPROVED** [review]
🙏 Thank you thank you thank you @ryley-o

File: scripts/util/aws_s3.ts [hunk 1]
```diff
@@ -20,14 +22,27 @@ const s3Client = new S3Client({
...
     ObjectOwnership: "ObjectWriter",
   };
   const command = new CreateBucketCommand(input);
   return await client.send(command);
+  // remove block public access
+  const input2 = {
+    Bucket: bucketName,
+  };
+  const command2 = new DeletePublicAccessBlockCommand(input2);
+  const response = await client.send(command);
```
Comment (lyaunzbe) [inline scripts/util/aws_s3.ts:38]: this needs to be command2

File: scripts/util/aws_s3.ts [hunk 2]
```diff
@@ -20,14 +22,27 @@ const s3Client = new S3Client({
...
+  // set bucket ACL to public-read
+  const input3 = {
+    Bucket: bucketName,
+    ACL: "public-read",
+  };
+  const command3 = new PutBucketACLCommand(input3);
+  return await client.send(command);
```
Comment (lyaunzbe) [inline scripts/util/aws_s3.ts:45]: needs to be command3

**Review by lyaunzbe — APPROVED** [review]

**Review by lyaunzbe — CHANGES_REQUESTED** [review]
woops meant to request on the prev review

File: scripts/util/aws_s3.ts (hunk 1 above)
Comment (ryley-o) [reply scripts/util/aws_s3.ts]: oops, thanks and updated!

File: scripts/util/aws_s3.ts (hunk 2 above)
Comment (ryley-o) [reply scripts/util/aws_s3.ts]: oops, thanks and updated!

**Review by lyaunzbe — APPROVED** [review]

**Pull request merged**